tests/e2e/tests/mixer/mixer_test.go

```diff
@@ -463,16 +474,31 @@ func errorf(t *testing.T, format string, args ...interface{}) {
 }
 
 func TestMetric(t *testing.T) {
-	checkMetricReport(t, destLabel, fqdn("productpage"))
+	checkMetricReport(t, "istio_requests_total", destLabel, fqdn("productpage"))
 }
 
 func TestIngressMetric(t *testing.T) {
-	checkMetricReport(t, srcWorkloadLabel, "istio-"+ingressName)
+	checkMetricReport(t, "istio_requests_total", srcWorkloadLabel, "istio-"+ingressName)
+}
+
+func TestOOPMetric(t *testing.T) {
+	if err := applyMixerRule(oopPromConfigs); err != nil {
+		fatalf(t, "could not apply required out of process configs: %v", err)
+	}
+
+	defer func() {
+		if err := deleteMixerRule(oopPromConfigs); err != nil {
+			t.Logf("could not clear out of process config: %v", err)
+		}
+	}()
+	allowRuleSync()
+
+	checkMetricReport(t, "istio_request_count_oop", destLabel, fqdn("productpage"))
```

Comment:
Do I understand correctly that this measure the number of OOP calls? If so, I'd like to see some functional check as well. Maybe call some application endpoint and measure application metric?

Reply:
No, this is a replication of default request count metric: https://github.com/istio/istio/pull/12577/files#diff-42f57e2875ff647fa7593c2fc4675c53R44. So this is actually a functional check.

Reply:
OK thanks for clarifying.